src/test/groovy/page/DashboardPage.groovy

```diff
@@ -9,15 +9,15 @@ class DashboardPage extends Page {
 
     static at = {
         title == "- ExamPlanner"
-        registeredTable.displayed
-        calendar.displayed
+        $("#productList").displayed
```

Comment:
Krzysiek chciał, żebym poprawił to za niego. Mówił, że zmiennych z "content" nie można używać w definicji "at".